models/kyberswap/arbitrum/kyberswap_aggregator_arbitrum_trades_legacy.sql 

```diff
@@ -0,0 +1,9 @@
+{{ config
+(
+    tags=['legacy'],
+    alias = alias('aggregator_trades', legacy_model=True),
```

Comment:
There is a extra `,` at the end of this line:
```suggestion
    alias = alias('aggregator_trades', legacy_model=True)
```

Reply:
Fixed.